packages/web/components/complex/portfolio/assets-overview.tsx

```
@@ -265,40 +265,51 @@ export const AssetsOverview: FunctionComponent<
               />
-              <span className="subtitle1">{t("portfolio.buy")}</span>
+              <span className="subtitle1 md:subtitle2">
+                {t("portfolio.buy")}
+              </span>
             </Button>
             <Button
-              className="flex h-[48px] !w-[141px] items-center gap-2 !rounded-full !bg-osmoverse-825 !p-0 text-wosmongton-200 hover:!bg-osmoverse-800"
+              className="flex h-[48px] items-center gap-2 !rounded-full !bg-osmoverse-825 text-wosmongton-200 hover:!bg-osmoverse-800 md:gap-1"
+              size={isMobile ? "xsm" : undefined}
               onClick={() => startBridge({ direction: "withdraw" })}
               disabled={totalValue && totalValue?.toDec()?.isZero()}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)